examples/pendulum/pendulum.bioMod

```diff
@@ -0,0 +1,30 @@
+version 3
+gravity 0 0 -9.81
+
+
```

Comment:
Could you rename the folder getting_started (keep pendulum for both pendulum.bioMod and .py). This straightforward example should be the first step for a new comer